src/fixtures/FixtureCache.js

```diff
@@ -1,12 +1,12 @@
-const _bindAll = require('lodash.bindall');
-
 module.exports = function (Promise) {
   class FixtureCache {
 
     constructor() {
       this.cache = {};
 
-      _bindAll(this, ['set', 'get', 'getOrPromise', 'setAsync']);
```

Comment:
getOrPromise is not called in an async method in the class, so context is not needed